src/v2/Apps/Order/Routes/Payment/index.tsx

```diff
@@ -229,6 +204,11 @@ export const PaymentFragmentContainer = createFragmentContainer(
             node {
               artwork {
                 slug
+                partner {
+                  name
+                  # TODO: when it is available, pull below field to check if wire T enabled for partner
+                  # wireTransferEnabled
+                }
```

Comment:
@starsirius left a [suggestion](https://github.com/artsy/metaphysics/pull/4065#discussion_r876006119) on the PR that @kiryl-zubarau opened to include this flag in Metaphysics. Depending on how that goes, this field could be available on an `order` instead of a `partner`.